src/effects/VST/VSTEffect.cpp

```diff
@@ -2131,6 +2160,23 @@ wxString VSTEffectWrapper::GetString(int opcode, int index) const
    return str;
 }
 
+wxString VSTEffectWrapper::GetStringFromHandle(int opcode, AEffect* handle) const
```

Comment:
`GetStringFromHandle` can become a static, nonmember function if `handle` is of the new structure type.